gcp_variant_transforms/testing/integration/small_tests/merge_option_info_keys_to_move_to_calls_regex.json

```diff
@@ -0,0 +1,54 @@
+{
+  "test_name": "merge-option-info-keys-to-move-to-calls-regex",
+  "table_name": "merge_option_info_keys_to_move_to_calls_regex",
+  "input_pattern": "gs://gcp-variant-transforms-testfiles/small_tests/merge/*.vcf",
+  "variant_merge_strategy": "MOVE_TO_CALLS",
+  "info_keys_to_move_to_calls_regex": "NS",
```

Comment:
nit: this should be `^NS$` as otherwise it would match anything that has "NS" as a substring.

Reply:
Done.